library/cartridge_instance.py

```diff
@@ -14,21 +15,115 @@
 }
 
 
+def serialize_dict(d):
+    parts = []
+
+    for k, v in d.items():
+        value_str = None
+        if type(v) == str:
+            value_str = "'%s'" % v
+        elif type(v) == float or type(v) == int:
+            value_str = "%s" % v
+        elif type(v) == bool:
+            value_str = 'true' if v else 'false'
+        else:
+            raise CartridgeException(
```

Comment:
It doesn't looks like "CartridgeException". E.g. "SerializeException"... but it's not related to cartridge

Reply:
It's called `CartridgeException` for historical reasons, all such exceptions aren't related to cartridge.